dydx-testnet-4/gentx/POPSTeam.json

```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"P-OPS Team","identity":"A2B9CAA088770EA6","website":"https://www.pops.one","security_contact":"security@pops.one","details":"P-OPS TEAM is a decentralized organization providing you with validation and staking services, blockchain consultation, growth acceleration and investment capital for innovative Web 3.0 projects."},"commission":{"rate":"0.050000000000000000","max_rate":"0.200000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"dydx1g2a066hzvw2kju59zlkc3mfnc5lvvx6vhhvjm0","validator_address":"dydxvaloper1g2a066hzvw2kju59zlkc3mfnc5lvvx6vjf6ys7","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"zcdhOSyqsyZryTi6pCk5s9dlLlNaCYfEDmc5HK6w+Bw="},"value":{"denom":"dv4tnt","amount":"50000000000"}}],"memo":"27623a574fabf63dc0ea5cb1a294523c17c6d36e@188.42.220.52:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A9/2vx4fZWJvi31zIeToDCIyFt8Syh9TkaCg9zGPYmJ9"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""},"tip":null},"signatures":["Ly41Dr+nZt2d7HvfdUjYqICHAOK2Csi83RnKkDVoeIMIotomhYjR5O+XsJlXOaaMxe56aAZVnp4/fmdpaqpk/A=="]}
```

Comment:
the [Prepare for Genesis](https://v4-teacher.vercel.app/validators/preparing_for_genesis) guide is a little different this time, the test token is adv4tnt and the self-delegation amount is 500000000000000000000000adv4tnt

Reply:
ok done that also, sorry for the delay

Reply:
no worries!